clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -229,6 +231,19 @@ class CIRGenBuilderTy : public cir::CIRBaseBuilderTy {
   cir::IntType getUInt32Ty() { return typeCache.UInt32Ty; }
   cir::IntType getUInt64Ty() { return typeCache.UInt64Ty; }
 
+  cir::ConstantOp getConstInt(mlir::Location loc, llvm::APSInt intVal);
+
+  cir::ConstantOp getConstInt(mlir::Location loc, llvm::APInt intVal);
+
+  cir::ConstantOp getConstInt(mlir::Location loc, mlir::Type t, uint64_t c);
+
+  cir::ConstantOp getConstFP(mlir::Location loc, mlir::Type t,
+                             llvm::APFloat fpVal) {
+    assert(mlir::isa<cir::CIRFPTypeInterface>(t) &&
+           "expected floating point type");
+    return create<cir::ConstantOp>(loc, getAttr<cir::FPAttr>(t, fpVal));
+  }
```

Comment:
The implementation of `getConstInt` and `getConstFP` looks very similar, yet `getConstInt` is defined in `CIRGenBuilder.cpp` while `getConstFP` is defined in the header. Can we make it consistent?